src/CMakeLists.txt

```diff
@@ -16,6 +16,8 @@
 if(NOT MSVC)
     add_compile_options(-Wall -W -Wextra)
     add_compile_options(-Wno-ignored-attributes)
+else()
+	set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
```

Comment:
When creating libraries on windows it is necessary to explicitly state, which symbols should be exported using `__declspec(dllexport)`. This is usually wrapped in a macro like `CLVK_EXPORT` and then added to all functions that should be exported by the dll.
The quick _fix_ is to export all symbols, this behavior is triggered by this CMake variable.

Though, I'm open to go through clvk and add the `CLVK_EXPORT` a bit more selectively.. just wanted to push the diff before going to bed..

Reply:
I had a feeling this was the case :). Thanks!

>Though, I'm open to go through clvk and add the CLVK_EXPORT a bit more selectively.. just wanted to push the diff before going to bed..

That would be amazing 🥇 . Let me know if you're going for this or if you'd prefer getting this PR in first.

Reply:
I'll get this in as is since it's immediately useful. I've created https://github.com/kpet/clvk/issues/177 to cover making exports a bit more selective.